### DISCUSSION TITLE
BroadcastsEvents does not fire update event on polymorphic relationship update

### DISCUSSION THREAD

**Discussion by BaasNiel** [laravel/framework#38752] [Q&A]
I have a `Post` class with a polymorphic `Comment` exactly as defined in [Laravel's documentation](https://laravel.com/docs/8.x/eloquent-relationships#one-to-many-polymorphic-relations)

The `Post` class also has the `BroadcastsEvents` trait, and the `Comment` has `protected $touches = ['commentable']`.

I've noticed that the `Post` gets its `updated_at` property updated when a new comment gets added, but there is no `PostUpdated` event that is fired through `BroadcastsEvents`. 

Is this intended or am I doing something wrong?

**Comment by HenrijsS** [comment]
Hey.

Running in to this same exact problem.
Did you ever manage to fix it? :D
